crates/matrix-sdk-indexeddb/src/state_store/mod.rs

```diff
@@ -418,6 +418,9 @@ impl IndexeddbStateStore {
             StateStoreDataKey::Filter(filter_name) => {
                 self.encode_key(key.encoding_key(), (key.encoding_key(), filter_name))
             }
+            StateStoreDataKey::UserAvatarUrl(user_id) => {
+                self.encode_key(key.encoding_key(), (key.encoding_key(), user_id))
```

Comment:
I just talked about this encoding key thing with Kévin and realized we don't need to use it here for the first parameter. 

```suggestion
                self.encode_key(KEYS::KV, (key.encoding_key(), user_id))
```

Looks weird but I'll add a comment in the next PR to explain.